Server-Protocol-Buffer/src/main/proto/internal/ppdd/ppac_android.proto

```diff
@@ -3,9 +3,11 @@ syntax = "proto3";
 package de.rki.coronawarnapp.server.protocols.internal.ppdd;
 
 
+option java_outer_classname = "PPACAndroidProto";
```

Comment:
Well the issue is that this renames the generated protobuf files.

Reply:
I was hoping this is actually only relevant for Java... i'll fix it